components/Ui/UiCard.vue

```diff
@@ -187,30 +182,31 @@ function hasTags(tags: string[]) {
     display: flex;
     justify-content: space-between;
     align-items: flex-start;
-    margin-bottom: carbon.$spacing-03;
+    margin-bottom: carbon.$spacing-07;
 
     @include carbon.breakpoint-down(lg) {
       flex-direction: column;
     }
-
-    .cds--row {
-      margin-right: 0;
-      margin-left: 0;
-      margin-bottom: carbon.$spacing-03;
-    }
   }
 
   &__tag {
-    white-space: nowrap;
+    margin-right: carbon.$spacing-03;
+    background-color: qiskit.$tag-background-color;
+    color: qiskit.$tag-text-color;
+
+    &:first-child {
```

Comment:
Instead of styling with first child, we can have the modifier `__tag_primary` and give styles with that. In the future, if we have more than one primary tag, we won't have to modify more things

Reply:
I ended up using the `__tag_primary` modifier, which I needed to style the colors (background purple w/ white text)

However, we still need this first-child selector to properly align the tag the most left tag wit the title above, and text below

Reply:
Wait, I'm realizing this still needs an update... 

Why?
Because if there are no `primary-tags`, but there are `secondary-tags`, the left-margin still needs to be accounted for...

Reply:
addressed in [this commit](https://github.com/Qiskit/qiskit.org/pull/3442/commits/63d807865cd33ca4de306f89a3a0ea786ebb7ede)